tests/nosetests/pyanaconda_tests/module_dasd_test.py

```diff
@@ -42,6 +42,9 @@ def setUp(self):
         self.dasd_module = DASDModule()
         self.dasd_interface = DASDInterface(self.dasd_module)
 
+    def is_supported_test(self):
+        self.assertEqual(self.dasd_interface.IsSupported(), False)
```

Comment:
What will happen if this test will be started on S390?

Reply:
Many of our tests will probably fail on s390x. I can mock the architecture for this test, but I don't think our unit tests are ready to run on a different architecture in general.

Reply:
Anyway, I have fixed the tests.

Reply:
Thanks for fixing this. We want to move these tests to gating tests so it would be great to make them robust if possible.